.github/workflows/ci.yml

```diff
@@ -28,9 +28,6 @@ jobs:
       # Install Flutter
       - name: Install Flutter
         uses: subosito/flutter-action@v2
-        with:
-          flutter-version: "3.0.5"
-          channel: "stable"
```

Comment:
@tk-nguyen we need to have a specific version, am I right?

Reply:
Yep.

Reply:
IMO, If we don't specify it, it will use the latest stable version

Reply:
If that's what you want, it's fine. It will use the latest version every time it runs, so keep that in mind.